src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/BackgroundDocumentGenerator.cs

```diff
@@ -250,7 +258,7 @@ private void NotifyDocumentsProcessed(List<WorkResult> results)
         }
     }
 
-    private void ProjectSnapshotManager_Changed(object sender, ProjectChangeEventArgs args)
+    private void ProjectSnapshotManager_Changed(object? sender, ProjectChangeEventArgs args)
```

Comment:
I assume you only made `sender` nullable because you had to but I'm kinda shocked at the idea that it'd be null, what's the scenairo?

Reply:
`EventHandler` and `EventHandler<TEventArgs>` make `sender` as `object?` because it's not uncommon for them to be used that way. Now that our projects multi-target and include `net7.0` as a target, we get those annotations and need to adhere to them. There's no scenario for us in Razor here, I don't think.